File: MUNityAngular/Models/ResolutionAdvancedInfoModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MUNityAngular.Models
{
    public class ResolutionAdvancedInfoModel : ResolutionInformationModel
    {
        public string OnlineCode { get; set; }

        public bool PublicRead { get; set; }

        public bool PublicWrite { get; set; }

        public bool PublicAmendment { get; set; }

        public DateTime CreationDate { get; set; }

        public DateTime LastChangedDate { get; set; }
    }
}

File: MUNityAngular/ClientApp/src/app/models/resolution/resolution-advanced-info.model.ts
import { ResolutionInformation } from "./resolution-information.model";

export class ResolutionAdvancedInfo extends ResolutionInformation {
  public OnlineCode: string;

  public PublicRead: boolean;

  public PublicWrite: boolean;

  public PublicAmendment: boolean;

  public CreationDate: Date;

  public LastChangedDate: Date;

}
